src/gui/accountmodalwidget.h

```diff
@@ -27,10 +28,19 @@ class AccountModalWidget : public QWidget
 public:
     AccountModalWidget(const QString &title, QWidget *widget, QWidget *parent);
 
+    enum DialogCode { Rejected, Accepted };
```

Comment:
enum class. and q_emum please.
Would Result be a better name?

Reply:
done